test/customized_container_test.py

```diff
@@ -27,6 +28,7 @@ def test_environment(host):
     assert "MYSQL_ENV_MYSQL_USER=test_user" in env
     assert "MYSQL_ENV_MYSQL_DATABASE=test_db" in env
     assert "MYSQL_ENV_MYSQL_PASSWORD=test_password" in env
+    assert "BACKUP_NAME=example.com" in env
     assert "BACKUP_TIME=1 2 3 4 5" in env
     assert "CLEANUP_OLDER_THAN=100" in env
 
```

Comment:
It is great to see, you also cared for the tests. Unfortunately you forgot to assert that the BACKUP_NAME is passed to the crontab command, so both "test_crontab" tests fail (in customized_container_test and minimal_settings_tests)

Reply:
The mentioned tests are still failing. Please run the test script `./test.sh` before pushing.